tabs/firmware_flasher.js

```diff
@@ -27,6 +27,10 @@ TABS.firmware_flasher.initialize = function (callback) {
         // translate to user-selected language
         localize();
 
+        function enable_load_online_button() {
+            $("a.load_remote_file").text(chrome.i18n.getMessage('firmwareFlasherButtonLoadOnline')).removeClass('disabled');
```

Comment:
element type selector is redundant, there is no reason to keep `a` at all in here, it only makes selector slower

Reply:
Just copied the existing selector already used by the code. I don't think performance will be a problem in this case and since we're selecting in class rather than id, I think the `a` at the front makes it less likely to screw up in the future.

Reply:
class and id has now almost the same performance. Could you just make me this pleasure and remove this ugly `a` please?

Reply:
Sure, no problem!

Reply:
Just to be clear... That `a` or all the occurrences of `a.load_remote_file`? There are 4 or 5 already in the file.

Reply:
The ones you touched and I will pretend I did not saw the rest :)